Simplified/NYPLBookAcquisitionPath.m

```diff
@@ -2,7 +2,6 @@
 
 #import "NYPLBookAcquisitionPath.h"
 
-NSString * const _Nonnull ContentTypeOverdriveCatalog = @"application/atom+xml;type=entry;profile=opds-catalog";
```

Comment:
I think this is being removed because it's redundant -- it's the same media type as `ContentTypeOPDSCatalog`. Is this right?

Reply:
Yeah, I added it when I first worked on Overdrive and didn't realize it's duplicated.